prow/apis/prowjobs/v1/types.go

```diff
@@ -71,6 +72,8 @@ const (
 	JenkinsAgent ProwJobAgent = "jenkins"
 	// KnativeBuildAgent means prow will schedule the job via a build-crd resource.
 	KnativeBuildAgent ProwJobAgent = "knative-build"
+	// TektonAgent means prow will schedule the job via a tekton PipelineRun CRD resource.
+	TektonAgent = "tekton"
```

Comment:
tekton-pipeline? Seems weird for agent: tekton to require a pipeline_run_spec

Reply:
`tekton-pipeline` was the original name, but I can rename it back.

Reply:
I renamed the agent to `tekton-pipeline`.